wagtail_modeltranslation/tests/tests.py

```diff
@@ -94,15 +98,27 @@ def setUpClass(cls):
 
                 # 5. makemigrations
                 from django.db import connections, DEFAULT_DB_ALIAS
-                call_command('makemigrations', verbosity=2, interactive=False,
-                             database=connections[DEFAULT_DB_ALIAS].alias)
+                if VERSION[0] < 2:
```

Comment:
I'm curious about the changes to `call_command` invocation, has Wagtail 2.0 changed the API?

Reply:
I need to put this inside a check for django.

Later versions of django have changed the parameters that makemigrations and migrate support.   Have to say, I was kind of guessing what would work.

You can probably see some earlier travis builds where it was complaining about unsupported parameters.

Reply:
I see.